web/src/js/__tests__/ducks/optionsSpec.js

```diff
@@ -49,3 +49,18 @@ describe('sendUpdate', () => {
         ])
     })
 })
+
+describe('dump', () => {
```

Comment:
Pressing Escape does not save the file now, does it? I was also thinking about making the modal disappear by clicking on the background. Maybe we actually want to do this in `hideModal`, or OptionModal's `componentWillUnmount`? I totally see the motivation why you wanted to separate this, but that looks it leaves a lot of room for error.